verilog/tools/ls/autoexpand_test.cc

```diff
@@ -2377,8 +2379,8 @@ module foo (  /*AUTOARG*/
       .clk(clk),
       .rst(rst),
       // Outputs
-      .o1(out_a[63:0]),
-      .o2(out_b  /*.[16]*/)
+      .o1 (out_a[63:0]),
+      .o2 (out_b  /*.[16]*/)
   );
 endmodule
 
```

Comment:
If not already, it might be worthwhile testing a file with multiple autoexpand sections and some, 'freestyle' formatting by the user in-between these (that does not conform to the formatter), then apply auto expand.

What we then expect to see is

   * sections between the auto-expanded parts are still featuring the original formatting
   * the formatting does the right thing of not messing up regions - since the expanded areas are typically longer than before we still want to see that the resulting document looks nice.

Reply:
I get the first dot, but what do you mean by *not messing up regions*?

Reply:
_messing up regions_: since we're replacing sections of the input with new output that might be more lines, the whole lower part of the file changes down. Previously, you made sure to apply edits from the bottom to the top, but since now the formatter is involved which also looks at sub-ranges and emits other subranges, there is a chance that for whatever reason we create overlapping regions that 'eat' up output. At least this is something we should confirm with a test that it does not happen.

Reply:
Added a test for it. Seems like the formatter preserves some weird formatting in the ranges that AUTO expansion puts in declarations and connections (they are copied verbatim from the user's code).